### PR TITLE
fix images from doc

### PR DESCRIPTION
**Pull request by Fabious**
ref #50 

Images in markdown were not parsed properly by gatsby-remark because they were in a folder outside the said markdown file.

Installing this plugin (https://www.npmjs.com/package/gatsby-remark-relative-images) fix this issue.

As a bonus images are responsive and you can see them fullscreen when clicking on it (this can be disabled if necessary)

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -8,6 +8,33 @@ module.exports = {
...
+        path: `${__dirname}/src/pages`,
+        name: 'pages',
+      },
+    },
+    {
+      resolve: `gatsby-transformer-remark`,
+      options: {
+        plugins: [
+          {
+            resolve: `gatsby-remark-relative-images`,
```
Comment (alanpoulain) [inline gatsby-config.js]: Can't we use the short syntax since it has no option?

Comment (Fabious) [reply gatsby-config.js]: Yes we can 👍

**Comment by dunglas** [comment]
Thanks @Fabious

**Comment by dunglas** [comment]
It works https://api-platform.com/docs/core/serialization/#the-serialization-process 🎇

**Pull request merged**